Instruction to install alpha, beta, rc and nightly (#1891)
pydata/pydata-sphinx-theme: docs/community/setup.md

If you are comfortable with and prefer a more manual setup refer to the [](topics/manual-dev.md) section.

## Testing pre-release and nightly

> If you are uncomfortable with development, you can already test the alpha, beta

Reviewer: ```suggestion
You can test the alpha, beta
```

> and release candidate of pydata sphinx theme on your your projects. Do do so

Reviewer: ```suggestion
and release candidates of pydata sphinx theme on your your projects. To do so
```

simply install with pip using the `--pre` flag:

```console
$ pip install --pre pydata-sphinx-theme
```

If an `alpha`, `beta` or `rc` is available, pip will install it.

> You can use the `--pre` flag in your projects continuous integration test suite

Reviewer: ```suggestion
You can use the `--pre` flag in your project's continuous integration test suite
```

to catch regressions or bugs before their release.

> If you are even more adventurous pydata-sphinx-theme has nightly build, you can try following the

Reviewer: ```suggestion
If you are even more adventurous pydata-sphinx-theme has nightly builds, you can try following the
```

instructions provided [on the scientific-python/upload-nightly-action
Readme](https://github.com/scientific-python/upload-nightly-action?tab=readme-ov-file#using-nightly-builds-in-ci)

> on install nightly wheels.

Reviewer: ```suggestion
on installing nightly wheels.
```

> Again installing nightly wheel on CI jobs is recommended if you wish to help catching bugs ahead of

Reviewer: ```suggestion
Installing nightly wheels in your project's CI jobs is a great way to help theme developers catch bugs ahead of
```
